docs/src/api.md

```diff
@@ -100,6 +100,64 @@ ADC
 Delay
 ```
 
+## Functions for structs
+
+### `dur`
+```@docs
+dur
+```
```

Comment:
Are the multiple methods of `dur` shown here?

Reply:
Yes. They are displayed like so:
![image](https://github.com/cncastillo/KomaMRI.jl/assets/36004160/ec4ee61c-f115-46ec-92f9-13da3a8df857)